ephemera/src/core/ephemera.rs

```diff
@@ -120,7 +120,7 @@ impl<A: Application> Ephemera<A> {
         while !shutdown.is_shutdown() {
             tokio::select! {
                 biased;
-                _ = shutdown.recv() => {
+                () = shutdown.recv() => {
```

Comment:
I'm surprised clippy cares about that one. I guess you'll have plenty of those to fix now

Reply:
We're seeing it only in ephemera as it has pedantic enabled